Transition.cs

```diff
@@ -0,0 +1,34 @@
+using Godot;
+using System;
+using System.Threading.Tasks;
+
+public class Transition : Control
```

Comment:
This class basically takes an `Image`, and runs an animation on the shader to do pixelated in/out (see `pixelated` shader [here](https://github.com/fourlastor-jams/game-off-2022/pull/30/files?diff=split&w=1#diff-ac57e276f0b5f0fc20187150f5aa5562edcd245810447986174bcaaa7cf05117))